.github/workflows/conda-package.yml

```diff
@@ -432,6 +432,190 @@ jobs:
         env:
           ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
 
+  array-api-conformity:
+    name: Array API conformity
+
+    needs: build
+
+    permissions:
+      # Needed to add a comment to a pull request's issue
+      pull-requests: write
+
```

Comment:
Could you set a timeout in case there is a hang during test runs?

Reply:
It has to be a general improvement across all workflows, because currently we are not defining any timeout at all.